src/conduits/RwaMultiSwapOutputConduit.sol

```diff
@@ -397,9 +395,10 @@ contract RwaMultiSwapOutputConduit {
     function requiredDaiWad(uint256 amt) external view returns (uint256 wad) {
         require(psm != address(0), "RwaMultiSwapOutputConduit/psm-not-hooked");
 
-        uint256 amt18 = _mul(amt, 10**_sub(18, GemAbstract(gem()).decimals()));
-        uint256 fee = _mul(amt18, PsmAbstract(psm).tout()) / WAD;
-        return _add(amt18, fee);
+        uint256 amt18 = amt * 10**(18 - uint256(GemAbstract(gem()).decimals()));
```

Comment:
We should add a comment that we are ensuring `18 - uint256(GemAbstract(gem()).decimals()´ won't overflow before.